tests/tf-manifests/aws/add-account-roles/rosa-classic/main.tf

```diff
@@ -28,17 +28,10 @@ data "rhcs_versions" "all" {
 }
 
 module "create_account_roles" {
-  source  = "terraform-redhat/rosa-sts/aws"
-  version = ">= 0.0.14"
+  source  = "terraform-redhat/rosa-classic/rhcs//modules/account-iam-resources"
```

Comment:
do we need the `//` ?

Reply:
yeah. that's telling terraform to use a sub-module of the rhcs module